ansible/management-searchengine.yml

```diff
@@ -0,0 +1,253 @@
+# Search ngine + Search Engine Client + Elasticsearch
+
+#- hosts: "{{ idr_environment | default('idr') }}-management-hosts"
```

Comment:
Should this line be uncommented?

Reply:
This line has been uncommented and moved to replace local in  (host: local) and removed (connection: local) as I was testing them locally. I have done the same for the other playbooks. I think this should work fine?